From be8ec40c62b928c4b92e62ea9767df9a49ddba6b Mon Sep 17 00:00:00 2001
From: Carl George <carl@george.computer>
Date: Tue, 8 Aug 2023 13:15:12 -0500
Subject: [PATCH] Remove duplicate icons

Currently the same icons are installed to both /usr/share/icons/hicolor
and /usr/share/blackbox/icons/hicolor.  This change removes the latter.
---
 data/icons/meson.build | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/data/icons/meson.build b/data/icons/meson.build
index 6596c86..328522e 100644
--- a/data/icons/meson.build
+++ b/data/icons/meson.build
@@ -1,7 +1,4 @@
 icondir = join_paths(get_option('datadir'), 'icons/hicolor')
-pkgicondir = join_paths(get_option('datadir'), 'blackbox/icons/hicolor')
-
-install_subdir('hicolor/scalable', install_dir: pkgicondir)
 
 install_data(['hicolor/scalable/apps/com.raggesilver.BlackBox.svg'],
              install_dir: join_paths(icondir, 'scalable/apps'))
-- 
2.48.1

